packages/kube-client/lib/mixins.js

```diff
@@ -287,8 +287,12 @@ NamespaceScoped.Writable = superclass => class extends NamespaceScoped.Creatable
   }
 
   mergePatch (namespace, name, body, options) {
+    if (Array.isArray(name)) {
```

Comment:
I think it would be better to implement this inside of `assertName`:
```js
function assertName (name) {
  if (Array.isArray(name)) {
    name = name[0]
  }
  if (typeof name !== 'string' || !name) {
    throw new TypeError('The parameter "name" must be a string or an array of strings')
  }
}
```